src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        for (int i = 0; i < getUsers().length; i++) {
+            if (getUsers()[i].getEmail().equals(email)
+                    && getUsers()[i].getPassword().equals(password)) {
+                return true;
+            }
+        }
```

Comment:
I would suggest make use of `UserService.findByEmail` when implementing login method. We should not access users here directly or via getter.